app/views/DR/EpidemiologicalResponseScreen/epidemiologicalTips.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import PDFView from 'react-native-pdf';
+
+import Colors from '../../../constants/colors';
+import { GetStoreData, SetStoreData } from '../../../helpers/General';
+
+class MentalHealthAdvices extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      date: '',
+      tip: '',
+    };
+  }
+
+  getData = async () => {
+    try {
+      const storageData = await GetStoreData('epidemiologicalTips');
+
+      return storageData
+        ? JSON.parse(storageData)
+        : { date: undefined, tip: 0 };
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  saveData = async (newDate, newTip) => {
+    try {
+      await SetStoreData('epidemiologicalTips', {
+        date: newDate,
+        tip: newTip,
+      });
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  isNewDate = dateStorage => {
```

Comment:
Do really understand the purpose of this function,
Could you help me?

Reply:
Is for make a condition with today date and the date storage in AsycnStorage

Reply:
Ok